### ISSUE TITLE
View [mail.reset_password] not found.

### ISSUE THREAD

**Issue by XTREME1738** [anonaddy/docker#259]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/anonaddy/docker/blob/master/.github/SUPPORT.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

I am trying to reset my password for my self-hosted addy.io instance.
When trying I am getting an error every time, so I enabled APP_DEBUG.
From what I can see nothing important shows in the docker container logs.

### Expected behaviour

Email verification link to be sent to the account email.

### Actual behaviour

Receive either "500 | Server Error" screen if APP_DEBUG is false otherwise get "View [mail.reset_password] not found."
```
InvalidArgumentException:
View [mail.reset_password] not found.

  at /var/www/anonaddy/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137
  at Illuminate\View\FileViewFinder->findInPaths()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:79)
  at Illuminate\View\FileViewFinder->find()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/View/Factory.php:137)
  at Illuminate\View\Factory->make()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Mail/Markdown.php:65)
  at Illuminate\Mail\Markdown->render()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/Channels/MailChannel.php:114)
  at Illuminate\Notifications\Channels\MailChannel->Illuminate\Notifications\Channels\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Collections/helpers.php:224)
  at value()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Mail/Mailer.php:429)
  at Illuminate\Mail\Mailer->renderView()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Mail/Mailer.php:408)
  at Illuminate\Mail\Mailer->addContent()
     (/var/www/anonaddy/app/CustomMailDriver/CustomMailer.php:50)
  at App\CustomMailDriver\CustomMailer->send()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/Channels/MailChannel.php:66)
  at Illuminate\Notifications\Channels\MailChannel->send()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/NotificationSender.php:148)
  at Illuminate\Notifications\NotificationSender->sendToNotifiable()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/NotificationSender.php:106)
  at Illuminate\Notifications\NotificationSender->Illuminate\Notifications\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Support/Traits/Localizable.php:19)
  at Illuminate\Notifications\NotificationSender->withLocale()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/NotificationSender.php:101)
  at Illuminate\Notifications\NotificationSender->sendNow()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/ChannelManager.php:54)
  at Illuminate\Notifications\ChannelManager->sendNow()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/SendQueuedNotifications.php:119)
  at Illuminate\Notifications\SendQueuedNotifications->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36)
  at Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Container/Util.php:41)
  at Illuminate\Container\Util::unwrapIfClosure()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93)
  at Illuminate\Container\BoundMethod::callBoundMethod()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:35)
  at Illuminate\Container\BoundMethod::call()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Container/Container.php:662)
  at Illuminate\Container\Container->call()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:128)
  at Illuminate\Bus\Dispatcher->Illuminate\Bus\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:141)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:116)
  at Illuminate\Pipeline\Pipeline->then()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:132)
  at Illuminate\Bus\Dispatcher->dispatchNow()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php:123)
  at Illuminate\Queue\CallQueuedHandler->Illuminate\Queue\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:141)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:116)
  at Illuminate\Pipeline\Pipeline->then()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php:122)
  at Illuminate\Queue\CallQueuedHandler->dispatchThroughMiddleware()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php:70)
  at Illuminate\Queue\CallQueuedHandler->call()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php:102)
  at Illuminate\Queue\Jobs\Job->fire()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Queue/SyncQueue.php:43)
  at Illuminate\Queue\SyncQueue->push()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:254)
  at Illuminate\Bus\Dispatcher->pushCommandToQueue()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:230)
  at Illuminate\Bus\Dispatcher->dispatchToQueue()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php:77)
  at Illuminate\Bus\Dispatcher->dispatch()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/NotificationSender.php:229)
  at Illuminate\Notifications\NotificationSender->queueNotification()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/NotificationSender.php:76)
  at Illuminate\Notifications\NotificationSender->send()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/ChannelManager.php:39)
  at Illuminate\Notifications\ChannelManager->send()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Notifications/RoutesNotifications.php:18)
  at App\Models\User->notify()
     (/var/www/anonaddy/app/Models/User.php:423)
  at App\Models\User->sendPasswordResetNotification()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Auth/Passwords/PasswordBroker.php:72)
  at Illuminate\Auth\Passwords\PasswordBroker->sendResetLink()
     (/var/www/anonaddy/app/Http/Controllers/Auth/ForgotPasswordController.php:53)
  at App\Http\Controllers\Auth\ForgotPasswordController->sendResetLinkEmail()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Controller.php:54)
  at Illuminate\Routing\Controller->callAction()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php:43)
  at Illuminate\Routing\ControllerDispatcher->dispatch()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Route.php:259)
  at Illuminate\Routing\Route->runController()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Route.php:205)
  at Illuminate\Routing\Route->run()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Router.php:799)
  at Illuminate\Routing\Router->Illuminate\Routing\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:141)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/app/Http/Middleware/RedirectIfAuthenticated.php:28)
  at App\Http\Middleware\RedirectIfAuthenticated->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/inertiajs/inertia-laravel/src/Middleware.php:87)
  at Inertia\Middleware->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php:50)
  at Illuminate\Routing\Middleware\SubstituteBindings->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:78)
  at Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Session/Middleware/AuthenticateSession.php:41)
  at Illuminate\Session\Middleware\AuthenticateSession->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Middleware/ThrottleRequestsWithRedis.php:65)
  at Illuminate\Routing\Middleware\ThrottleRequestsWithRedis->handleRequest()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Middleware/ThrottleRequests.php:90)
  at Illuminate\Routing\Middleware\ThrottleRequests->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php:49)
  at Illuminate\View\Middleware\ShareErrorsFromSession->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:121)
  at Illuminate\Session\Middleware\StartSession->handleStatefulRequest()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php:64)
  at Illuminate\Session\Middleware\StartSession->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php:37)
  at Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php:67)
  at Illuminate\Cookie\Middleware\EncryptCookies->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:116)
  at Illuminate\Pipeline\Pipeline->then()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Router.php:798)
  at Illuminate\Routing\Router->runRouteWithinStack()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Router.php:777)
  at Illuminate\Routing\Router->runRoute()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Router.php:741)
  at Illuminate\Routing\Router->dispatchToRoute()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Routing/Router.php:730)
  at Illuminate\Routing\Router->dispatch()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:200)
  at Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:141)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:21)
  at Illuminate\Foundation\Http\Middleware\TransformsRequest->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php:31)
  at Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php:21)
  at Illuminate\Foundation\Http\Middleware\TransformsRequest->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TrimStrings.php:40)
  at Illuminate\Foundation\Http\Middleware\TrimStrings->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php:27)
  at Illuminate\Foundation\Http\Middleware\ValidatePostSize->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php:99)
  at Illuminate\Foundation\Http\Middleware\PreventRequestsDuringMaintenance->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Http/Middleware/HandleCors.php:49)
  at Illuminate\Http\Middleware\HandleCors->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Http/Middleware/TrustProxies.php:39)
  at Illuminate\Http\Middleware\TrustProxies->handle()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:180)
  at Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php:116)
  at Illuminate\Pipeline\Pipeline->then()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:175)
  at Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
     (/var/www/anonaddy/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php:144)
  at Illuminate\Foundation\Http\Kernel->handle()
     (/var/www/anonaddy/public/index.php:51)
```

### Steps to reproduce

1. Go to login
2. Click "Forgot Password?"
3. Enter username and captcha
4. Click "Send Verification Link"
5. Get Error

### Docker info

```text
Client: Docker Engine - Community
 Version:    25.0.3
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.12.1
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.24.5
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 5
  Running: 5
  Paused: 0
  Stopped: 0
 Images: 8
 Server Version: 25.0.3
 Storage Driver: overlay2
  Backing Filesystem: xfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: inactive
 Runtimes: runc io.containerd.runc.v2
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: ae07eda36dd25f8a1b98dfbf587313b99c0190bb
 runc version: v1.1.12-0-g51d5e94
 init version: de40ad0
 Security Options:
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 5.14.0-284.18.1.el9_2.x86_64
 Operating System: AlmaLinux 9.3 (Shamrock Pampas Cat)
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 10.7GiB
 Name: de1
 ID: 36831521-780c-4c9c-8d9b-73ccf597f6f0
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

### Docker Compose config

```yaml
name: addy
services:
  addy:
    container_name: addy
    depends_on:
      db:
        condition: service_started
        required: true
      redis:
        condition: service_started
        required: true
    environment:
      ANONADDY_ADDITIONAL_USERNAME_LIMIT: "3"
      ANONADDY_ADMIN_USERNAME: addy
      ANONADDY_ALL_DOMAINS: addyio.<redacted>
      ANONADDY_BANDWIDTH_LIMIT: "104857600"
      ANONADDY_DNS_RESOLVER: 127.0.0.1
      ANONADDY_DOMAIN: <redacted>
      ANONADDY_ENABLE_REGISTRATION: "true"
      ANONADDY_HOSTNAME: mail.<redacted>
      ANONADDY_LIMIT: "200"
      ANONADDY_NEW_ALIAS_LIMIT: "10"
      ANONADDY_RETURN_PATH: bounces@<redacted>
      ANONADDY_SECRET: <redacted>
      APP_DEBUG: "true"
      APP_KEY: <redacted>
      APP_URL: https://addyio.<redacted>
      DB_DATABASE: addyio
      DB_HOST: db
      DB_PASSWORD: <redacted>
      DB_USERNAME: addyio
      LOG_IP_VAR: remote_addr
      MAIL_FROM_ADDRESS: noreply@<redacted>
      MAIL_FROM_NAME: <redacted>
      MEMORY_LIMIT: 256M
      OPCACHE_MEM_SIZE: "128"
      PGID: "1000"
      POSTFIX_DEBUG: "true"
      POSTFIX_SMTP_TLS: "true"
      POSTFIX_SMTPD_TLS: "true"
      POSTFIX_SMTPD_TLS_CERT_FILE: /etc/letsencrypt/live/<redacted>/fullchain.pem
      POSTFIX_SMTPD_TLS_KEY_FILE: /etc/letsencrypt/live/<redacted>/privkey.pem
      PUID: "1000"
      REAL_IP_FROM: 0.0.0.0/32
      REAL_IP_HEADER: X-Forwarded-For
      REDIS_HOST: redis
      TZ: Europe/London
      UPLOAD_MAX_SIZE: 16M
    image: anonaddy/anonaddy:latest
    networks:
      default: null
    ports:
      - target: 25
        published: "25"
        protocol: tcp
      - target: 8000
        published: "8060"
        protocol: tcp
    restart: always
    volumes:
      - type: bind
        source: /etc/addy.io/data
        target: /data
        bind:
          create_host_path: true
      - type: bind
        source: /etc/letsencrypt
        target: /etc/letsencrypt
        bind:
          create_host_path: true
  db:
    command:
      - mysqld
      - --character-set-server=utf8mb4
      - --collation-server=utf8mb4_unicode_ci
    container_name: addy_db
    environment:
      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
      MYSQL_DATABASE: addyio
      MYSQL_PASSWORD: <redacted>
      MYSQL_USER: addyio
    image: mariadb:10.5
    networks:
      default: null
    restart: always
    volumes:
      - type: bind
        source: /etc/addy.io/db
        target: /var/lib/mysql
        bind:
          create_host_path: true
  redis:
    container_name: addy_redis
    image: redis:4.0-alpine
    networks:
      default: null
    restart: always
networks:
  default:
    name: addy_default
```

### Logs

```text
[postfix-config] virtual_destination_recipient_limit = $default_destination_recipient_limit
[postfix-config] virtual_extra_recipient_limit = $default_extra_recipient_limit
[postfix-config] virtual_gid_maps =
[postfix-config] virtual_initial_destination_concurrency = $initial_destination_concurrency
[postfix-config] virtual_mailbox_base =
[postfix-config] virtual_mailbox_domains = addyio.<redacted>,unsubscribe.addyio.<redacted>,mysql:/etc/postfix/mysql-virtual-alias-domains-and-subdomains.cf
[postfix-config] virtual_mailbox_limit = 51200000
[postfix-config] virtual_mailbox_lock = fcntl, dotlock
[postfix-config] virtual_mailbox_maps =
[postfix-config] virtual_minimum_delivery_slots = $default_minimum_delivery_slots
[postfix-config] virtual_minimum_uid = 100
[postfix-config] virtual_recipient_limit = $default_recipient_limit
[postfix-config] virtual_recipient_refill_delay = $default_recipient_refill_delay
[postfix-config] virtual_recipient_refill_limit = $default_recipient_refill_limit
[postfix-config] virtual_transport = anonaddy:
[postfix-config] virtual_transport_rate_delay = $default_transport_rate_delay
[postfix-config] virtual_uid_maps =
[cont-init.d] 15-config-postfix.sh: exited 0.
[cont-init.d] 50-svc-main.sh: executing... 

DB migration
   INFO  Nothing to migrate.  
Clear cache
   INFO  Application cache cleared successfully.  
   INFO  Configuration cached successfully.  
   INFO  Blade templates cached successfully.  
   INFO  Routes cached successfully.  
   INFO  Broadcasting queue restart signal.  
[cont-init.d] 50-svc-main.sh: exited 0.
[cont-init.d] 60-svc-rspamd.sh: executing... 
INFO: Rspamd service disabled.
[cont-init.d] 60-svc-rspamd.sh: exited 0.
[cont-init.d] 61-svc-postfix.sh: executing... 
[cont-init.d] 61-svc-postfix.sh: exited 0.
[cont-init.d] 80-svc-cron.sh: executing... 
Fixing crontabs permissions...
[cont-init.d] 80-svc-cron.sh: exited 0.
[cont-init.d] 99-clean.sh: executing... 
[cont-init.d] 99-clean.sh: exited 0.
[cont-init.d] ~-socklog: executing... 
[cont-init.d] ~-socklog: exited 0.
[cont-init.d] done.
[services.d] starting services
crond: crond (busybox 1.36.1) started, log level 8
2024/02/15 17:51:58 [notice] 722#722: using the "epoll" event method
2024/02/15 17:51:58 [notice] 722#722: nginx/1.24.0
2024/02/15 17:51:58 [notice] 722#722: OS: Linux 5.14.0-284.18.1.el9_2.x86_64
2024/02/15 17:51:58 [notice] 722#722: getrlimit(RLIMIT_NOFILE): 1073741816:1073741816
2024/02/15 17:51:58 [notice] 722#722: start worker processes
2024/02/15 17:51:58 [notice] 722#722: start worker process 747
2024/02/15 17:51:58 [notice] 722#722: start worker process 748
2024/02/15 17:51:58 [notice] 722#722: start worker process 749
2024/02/15 17:51:58 [notice] 722#722: start worker process 750
[services.d] done.
[15-Feb-2024 17:51:58] NOTICE: fpm is running, pid 727
[15-Feb-2024 17:51:58] NOTICE: ready to handle connections
postfix/postlog: starting the Postfix mail system
Feb 15 17:51:58 mail postfix/postfix-script[976]: starting the Postfix mail system
Feb 15 17:51:58 mail postfix/master[977]: daemon started -- version 3.8.4, configuration /etc/postfix
crond: USER anonaddy pid 981 cmd php /var/www/anonaddy/artisan schedule:run --no-ansi --no-interaction
   INFO  No scheduled commands are ready to run.  
172.20.0.1 - - [15/Feb/2024:17:52:06 +0000] "GET / HTTP/1.1" 302 398 "-" "Mozilla/5.0 (Windows NT 10.0; rv:109.0) Gecko/20100101 Firefox/115.0"
172.20.0.1 - - [15/Feb/2024:17:52:06 +0000] "GET /login HTTP/1.1" 200 1547 "-" "Mozilla/5.0 (Windows NT 10.0; rv:109.0) Gecko/20100101 Firefox/115.0"
172.20.0.1 - - [15/Feb/2024:17:52:09 +0000] "GET /password/reset HTTP/1.1" 200 1578 "https://addyio.<redacted>/login" "Mozilla/5.0 (Windows NT 10.0; rv:109.0) Gecko/20100101 Firefox/115.0"
172.20.0.1 - - [15/Feb/2024:17:52:10 +0000] "GET /captcha/mini?ccrysixR HTTP/1.1" 200 6739 "https://addyio.<redacted>/password/reset" "Mozilla/5.0 (Windows NT 10.0; rv:109.0) Gecko/20100101 Firefox/115.0"
172.20.0.1 - - [15/Feb/2024:17:52:15 +0000] "POST /password/email HTTP/1.1" 500 6622 "https://addyio.<redacted>/password/reset" "Mozilla/5.0 (Windows NT 10.0; rv:109.0) Gecko/20100101 Firefox/115.0"
[15-Feb-2024 17:52:36] NOTICE: Terminating ...
2024/02/15 17:52:36 [notice] 722#722: signal 15 (SIGTERM) received from 715, exiting
2024/02/15 17:52:36 [notice] 747#747: exiting
2024/02/15 17:52:36 [notice] 748#748: exiting
2024/02/15 17:52:36 [notice] 750#750: exiting
2024/02/15 17:52:36 [notice] 749#749: exiting
2024/02/15 17:52:36 [notice] 749#749: exit
[15-Feb-2024 17:52:36] NOTICE: exiting, bye-bye!
2024/02/15 17:52:36 [notice] 747#747: exit
2024/02/15 17:52:36 [notice] 750#750: exit
2024/02/15 17:52:36 [notice] 748#748: exit
2024/02/15 17:52:36 [notice] 722#722: signal 17 (SIGCHLD) received from 749
2024/02/15 17:52:36 [notice] 722#722: worker process 749 exited with code 0
2024/02/15 17:52:36 [notice] 722#722: signal 29 (SIGIO) received
2024/02/15 17:52:36 [notice] 722#722: signal 17 (SIGCHLD) received from 747
2024/02/15 17:52:36 [notice] 722#722: worker process 747 exited with code 0
2024/02/15 17:52:36 [notice] 722#722: worker process 750 exited with code 0
2024/02/15 17:52:36 [notice] 722#722: signal 29 (SIGIO) received
2024/02/15 17:52:36 [notice] 722#722: signal 17 (SIGCHLD) received from 748
2024/02/15 17:52:36 [notice] 722#722: worker process 748 exited with code 0
2024/02/15 17:52:36 [notice] 722#722: exit
```

**Comment by willbrowningme** [comment]
What version are you running - https://github.com/anonaddy/docker/releases?

**Comment by XTREME1738** [comment]
I am using the latest version available.

**Comment by willbrowningme** [comment]
Could you try running `docker compose exec addy anonaddy view:cache` to see if that solves it.

**Comment by XTREME1738** [comment]
Same error after running that.

**Comment by Dominion0815** [comment]
same here with addy docker v1.0.4. 

DEBUG:
```
View [mail.reset_password] not found.
...
Mailer->addContent() in[ /var/www/anonaddy/app/CustomMailDriver/CustomMailer.php ](file:///var/www/anonaddy/app/CustomMailDriver/CustomMailer.php#L50)(line 50)

            // to creating view based emails that are able to receive arrays of data.
            if (! is_null($callback)) {
                $callback($message);
            }
            $this->addContent($message, $view, $plain, $raw, $data);
            // If a global "to" address has been set, we will set that address on the mail
            // message. This is primarily useful during local development in which each
            // message should be delivered into a single mail address for inspection.
            if (isset($this->to['address'])) {
...
```

**Comment by willbrowningme** [comment]
You're right, this was a bug that I fixed in v1.0.5. I've just added a Pull Request to update this image to the latest v1.0.7.

**Comment by willbrowningme** [comment]
Please update to `v1.0.5` which fixes this, thanks @crazy-max.
